src/screens/Embed/Embed.style.js

```diff
@@ -156,7 +156,7 @@ export const EmbedBody = styled(Box)`
 export const EmbedHeaderWrapper = styled(Box)`
   ${PreviewHeaderStyles}
   width: 100%;
-  padding: 17px 14px;
+  padding: 13px 14px;
```

Comment:
this edit is unrelated to this PR. i noticed that an update that was recently made to the embed to make the icons fix caused the top/bottom borders to get cut off (image below). this padding edit fixes that.

![Screen Shot 2020-11-16 at 9 03 35 PM](https://user-images.githubusercontent.com/44076375/99336511-62337900-284f-11eb-9b08-e629a2c3bddb.png)